### DISCUSSION TITLE
Long term timelapse and restart

### DISCUSSION THREAD

**Discussion by GijsLok** [gopro/labs#175] [General]
My GP110 is loaded with code:  !NmTfWp.60!S!36000E!U!1R  Power external (ac) and no battery inside.
Card 512GB inside. Wifi connection and Cloud connection.

It start at day 1 and at the end of the script it uploads the video to the cloud and shutsdown. The next day it does not startup again. 
What can be the problem?

**Comment by dnewman-gpsw** [comment]
GP110?   HERO11?   There is a bug on HERO11, the cloud off load is waiting for a user prompt that is preventing the script continuing the next day.  Just started the investigation into this error, as this was a new prompt on the 2.01 firmware.  If you request auto clear, I think the prompt might not occur.

**Reply by GijsLok** [reply]
So if I use this code: !08:00NmTp.60!S!20280E!U!1R how do I add "auto clear'?

**Reply by dnewman-gpsw** [reply]
It is in the camera's menus, there isn't a QR Code for it.  Under Preferences -> Auto Upload -> Auto Clear.
